src/main/java/org/jenkinsci/test/acceptance/junit/JenkinsAcceptanceTestRule.java

```diff
@@ -38,6 +40,18 @@
  * @author Kohsuke Kawaguchi
  */
 public class JenkinsAcceptanceTestRule implements MethodRule { // TODO should use TestRule instead
+
+    private boolean enableInstallWizard;
```

Comment:
I do not like this is passed through the rule. You are injecting the controller anyway so you can configure that directly on the controller.